charts/pulsar/values.yaml

```diff
@@ -120,7 +120,9 @@ components:
   # broker
   broker: true
   # functions
-  functions: true
+  # WARNING! Before enabling functions, make sure that all of your users are trusted since functions run user code
+  # and the current security sandbox is not sufficient to protect against malicious code.
+  functions: false
```

Comment:
Is there any documentation for function security that they could be pointed to?

Reply:
No such documentation exists currently.